common/common.go

```diff
@@ -49,14 +49,25 @@ const (
 	ChallengeRequestedMandate ChallengeIndicator = "challenge_requested_mandate"
 )
 
+type DocumentType string
+
+const (
+	Passport             DocumentType = "passport"
+	NationalIdentityCard DocumentType = "national_identity_card"
+	DrivingLicense       DocumentType = "driving_license"
+	CitizenCard          DocumentType = "citizen_card"
+	ResidencePermit      DocumentType = "residence_permit"
+	ElectoralId          DocumentType = "electoral_id"
+)
+
 type AccountHolderIdentificationType string
 
 const (
-	Passport            AccountHolderIdentificationType = "passport"
-	DrivingLicence      AccountHolderIdentificationType = "driving_licence"
-	NationalId          AccountHolderIdentificationType = "national_id"
-	CompanyRegistration AccountHolderIdentificationType = "company_registration"
-	TaxId               AccountHolderIdentificationType = "tax_id"
+	AccountHolderPassport AccountHolderIdentificationType = "passport"
+	DrivingLicence        AccountHolderIdentificationType = "driving_licence"
+	NationalId            AccountHolderIdentificationType = "national_id"
+	CompanyRegistration   AccountHolderIdentificationType = "company_registration"
+	TaxId                 AccountHolderIdentificationType = "tax_id"
```

Comment:
We can merge this enum with DocumentType enum, as they have the same values in the API Reference. So, both Account structs and Issuing structs should use DocumentType.